packages/build/src/plugins_core/edge_functions/index.ts

```diff
@@ -40,14 +45,37 @@ const coreStep = async function ({
   netlifyConfig: any
   edgeFunctionsBootstrapURL?: string
 }) {
-  const { edge_functions: declarations = [] } = netlifyConfig
-  const { deno_import_map: userDefinedImportMap } = netlifyConfig.functions['*']
+  const { edge_functions: declarations = [], functions } = netlifyConfig
+  const { deno_import_map: userDefinedImportMap } = functions['*']
+  const importMapPaths: string[] = [userDefinedImportMap]
   const distPath = resolve(buildDir, distDirectory)
   const internalSrcPath = resolve(buildDir, internalSrcDirectory)
   const distImportMapPath = join(dirname(internalSrcPath), IMPORT_MAP_FILENAME)
   const srcPath = srcDirectory ? resolve(buildDir, srcDirectory) : undefined
-  const sourcePaths = [internalSrcPath, srcPath].filter(Boolean) as string[]
-  logFunctions({ internalSrcDirectory, internalSrcPath, logs, srcDirectory, srcPath })
+  const frameworksAPISrcPath = resolve(buildDir, packagePath || '', FRAMEWORKS_API_EDGE_FUNCTIONS_ENDPOINT)
+
+  let generatedFunctionsPath = internalSrcPath
+
+  if (featureFlags.netlify_build_frameworks_api) {
+    if (await pathExists(frameworksAPISrcPath)) {
+      generatedFunctionsPath = frameworksAPISrcPath
+    }
+
+    const frameworkImportMap = resolve(
```

Comment:
We now look for import maps on a specific location, as opposed to reading that from the legacy `manifest.json` file.

https://www.notion.so/netlify/Frameworks-API-documentation-b35d9fd4efcd4979a520b0607907001e?pvs=4#499d17a1cfbb4c8da316c2216b51bf77